config/crd/bases/services.k8s.aws_adoptedresources.yaml

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
The real real problem here is the controller-gen version we used to generate those files (`v0.7.0`) - Check line 7 in both files. This bug was fixed in [`v0.8.0`](https://github.com/kubernetes-sigs/controller-tools/releases/tag/v0.8.0) (PR https://github.com/kubernetes-sigs/controller-tools/pull/517).
Can please regenerate these file using `v0.9.2` ? it's the version we use for the generated controller now.

Reply:
Script to regen the CRDs is here https://github.com/aws-controllers-k8s/runtime/blob/main/scripts/build-apis.sh

Reply:
Sorry, not sure I fully understand: I'm using `aws-controllers-k8s/code-generator@20e82a5` with `aws-controllers-k8s/runtime@007e038` (i.e. `latest`) to generate the files. From the tags those commits are way ahead of `v0.[8|9].x`?

Reply:
And the weird thing is that the other service-related CRDs are created without the newline.

Reply:
Yep the service related CRD are generated using controller-gen v0.9.2 - the common CRDs were generated long time ago using v0.7.0.
The thing is that CRD files are generated using `controller-gen` not `ack-generate`. 

Script that build service related CRDs: https://github.com/aws-controllers-k8s/code-generator/blob/main/scripts/build-controller.sh#L196-L200
Script that build common CRDs: https://github.com/aws-controllers-k8s/runtime/blob/main/scripts/build-apis.sh#L20-L22